cypress/integration/home.spec.ts

```diff
@@ -1,7 +1,7 @@
-it('has working link to comparator page', () => {
+it('should have a working link to comparator page', () => {
   cy.visit('/')
 
-  cy.contains('Try me now!').click()
+  cy.findByRole('button', { name: 'Try me now!' }).click()
```

Comment:
Oh that `findByRole` thing is nice!

Reply:
Right? I highly recommend you going through [Testing Library queries priority](https://testing-library.com/docs/queries/about#priority) (you don't need to worry about `getBy*` and  `queryBy*` variants since they don't exist in Cypress Testing Library).

Basically we want to query things as closer to the user as possible, so we want to avoid querying by id of the elements, using `ByRole` most of the time. `ByLabelText` is a common one when working with forms.

Reply:
Made a issue to track this reading work, so I can read it when I have time. 😉 

I still need to get a development environment setup (again), to be able to try out Cypress and migrate the tests I had made earlier but using the testing library more... 😄